### PR TITLE
Speech GA - library update

### PR DESCRIPTION
**Pull request by nirupa-kumar**
@nnegrey Please review.
speech_transcribe_auto_punctuation
speech_transcribe_enhanced_model

### CODE REVIEW

File: speech/beta/src/main/java/com/example/speech/Recognize.java [hunk 1]
```diff
@@ -0,0 +1,582 @@
...
+public class Recognize {
+
+  /** Run speech recognition tasks. */
+  public static void main(String... args) throws Exception {
+    if (args.length < 1) {
+      System.out.println("Usage:");
+      System.out.printf(
+          "\tjava %s \"<command>\" \"<path-to-image>\"\n"
+              + "Commands:\n"
+              + "\t model-selection | stream-punctuation | enhanced-model| metadata  \n"
```
Comment (nnegrey) [inline speech/beta/src/main/java/com/example/speech/Recognize.java]: enhanced-model should not be in beta, right?

Comment (nnegrey) [reply speech/beta/src/main/java/com/example/speech/Recognize.java]: Same for `stream-punctuation`

File: speech/beta/pom.xml [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+  <packaging>jar</packaging>
+
+  <!--
+    The parent pom defines common style checks and testing strategies for our samples.
+    Removing or replacing it should not affect the execution of the samples in anyway.
+  -->
+  <parent>
+    <groupId>com.google.cloud.samples</groupId>
+    <artifactId>shared-configuration</artifactId>
+    <version>1.0.9</version>
```
Comment (nnegrey) [inline speech/beta/pom.xml]: May need to bump this to `1.0.10`.

File: speech/beta/pom.xml [hunk 3]
```diff
@@ -0,0 +1,139 @@
+<!--
+  Copyright 2018, Google Inc.
```
Comment (nnegrey) [inline speech/beta/pom.xml]: Google LLC

Comment (nnegrey) [reply speech/beta/pom.xml]: Only in the newly created beta files, updates to existing files can be left as is.

File: speech/beta/src/main/java/com/example/speech/Recognize.java [hunk 4]
```diff
@@ -0,0 +1,582 @@
...
+
+      RecognizeResponse recognizeResponse = speech.recognize(recConfig, recognitionAudio);
+      // Just print the first result here.
+      SpeechRecognitionResult result = recognizeResponse.getResultsList().get(0);
+      // There can be several alternative transcripts for a given chunk of speech. Just use the
+      // first (most likely) one here.
+      SpeechRecognitionAlternative alternative = result.getAlternativesList().get(0);
+      System.out.printf("Transcript : %s\n", alternative.getTranscript());
+    }
+    // [END speech_transcribe_model_selection_beta]
```
Comment (nnegrey) [inline speech/beta/src/main/java/com/example/speech/Recognize.java]: Let's update this to be outside the closing function bracket.

File: speech/beta/src/main/java/com/example/speech/Recognize.java [hunk 5]
```diff
@@ -0,0 +1,582 @@
...
+      List<SpeechRecognitionResult> results = response.get().getResultsList();
+
+      // Just print the first result here.
+      SpeechRecognitionResult result = results.get(0);
+      // There can be several alternative transcripts for a given chunk of speech. Just use the
+      // first (most likely) one here.
+      SpeechRecognitionAlternative alternative = result.getAlternativesList().get(0);
+      System.out.printf("Transcript : %s\n", alternative.getTranscript());
+    }
+    // [END speech_transcribe_model_selection_gcs_beta]
```
Comment (nnegrey) [inline speech/beta/src/main/java/com/example/speech/Recognize.java]: Same for this one.

File: speech/beta/src/main/java/com/example/speech/Recognize.java [hunk 6]
```diff
@@ -0,0 +1,582 @@
...
+            alternative.getWords((alternative.getWordsCount() - 1)).getSpeakerTag(),
+            alternative.getTranscript());
+      }
+    }
+  }
+
+  // [END speech_transcribe_diarization_gcs_beta]
+
+  // [START speech_transcribe_multichannel_beta]
+
```
Comment (nnegrey) [inline speech/beta/src/main/java/com/example/speech/Recognize.java]: Remove the empty line, (gets rid of a blank line in the docs)

File: speech/beta/src/main/java/com/example/speech/Recognize.java [hunk 7]
```diff
@@ -0,0 +1,582 @@
...
+        SpeechRecognitionAlternative alternative = result.getAlternatives(0);
+        System.out.format("Transcript : %s\n", alternative.getTranscript());
+        System.out.printf("Channel Tag : %s\n\n", result.getChannelTag());
+      }
+    }
+  }
+  // [END speech_transcribe_multichannel_beta]
+
+  // [START speech_transcribe_multichannel_gcs_beta]
+
```
Comment (nnegrey) [inline speech/beta/src/main/java/com/example/speech/Recognize.java]: Remove the empty line.

File: speech/beta/src/main/java/com/example/speech/Recognize.java [hunk 8]
```diff
@@ -0,0 +1,582 @@
...
+        // Print out the result
+        System.out.printf("Transcript : %s\n", alternative.getTranscript());
+        System.out.printf("Channel Tag : %s\n\n", result.getChannelTag());
+      }
+    }
+  }
+  // [END speech_transcribe_multichannel_gcs_beta]
+
+  // [START speech_transcribe_multilanguage_beta]
+
```
Comment (nnegrey) [inline speech/beta/src/main/java/com/example/speech/Recognize.java]: Remove the empty line.

File: speech/beta/src/main/java/com/example/speech/Recognize.java [hunk 9]
```diff
@@ -0,0 +1,582 @@
...
+        // first (most likely) one here.
+        SpeechRecognitionAlternative alternative = result.getAlternatives(0);
+        System.out.format("Transcript : %s\n\n", alternative.getTranscript());
+      }
+    }
+  }
+  // [END speech_transcribe_multilanguage_beta]
+
+  // [START speech_transcribe_multilanguage_gcs_beta]
+
```
Comment (nnegrey) [inline speech/beta/src/main/java/com/example/speech/Recognize.java]: Remove the empty line.

File: speech/beta/src/main/java/com/example/speech/Recognize.java [hunk 10]
```diff
@@ -0,0 +1,582 @@
...
+        // Print out the result
+        System.out.printf("Transcript : %s\n\n", alternative.getTranscript());
+      }
+    }
+  }
+  // [END speech_transcribe_multilanguage_gcs_beta]
+
+  // [START speech_transcribe_word_level_confidence_beta]
+
```
Comment (nnegrey) [inline speech/beta/src/main/java/com/example/speech/Recognize.java]: Remove the empty line.

File: speech/beta/src/main/java/com/example/speech/Recognize.java [hunk 11]
```diff
@@ -0,0 +1,582 @@
...
+        System.out.format(
+            "First Word and Confidence : %s %s \n",
+            alternative.getWords(0).getWord(), alternative.getWords(0).getConfidence());
+      }
+    }
+  }
+  // [END speech_transcribe_word_level_confidence_beta]
+
+  // [START speech_transcribe_word_level_confidence_gcs_beta]
+
```
Comment (nnegrey) [inline speech/beta/src/main/java/com/example/speech/Recognize.java]: Empty line.

**Comment by nirupa-kumar** [comment]
@nnegrey Added the additional sample. Please review and let me know if there is anything else.

**Pull request merged**